lib/trento/infrastructure/messaging/adapters/amqp.ex

```diff
@@ -9,8 +9,13 @@ defmodule Trento.Messaging.Adapters.AMQP do
 
   require Logger
 
+  # FIXME: fix Trento.Contracts.to_event/2
```

Comment:
What needs to be fixed in the contracts?

Reply:
There is an issue with a signature there. It is due abacus, I don't know how important it is, as we will most probably remove it. Anyway, we have tech debt ticket in our backlog to track this issue